templates/docs/base/code.md

```diff
@@ -18,20 +18,19 @@ When you refer to code inline with other text, use the <code>&lt;code></code> ta
 View example of inline code
 </a></div>
 
-### Block
```

Comment:
I don't think we should remove the base style documentation and examples.

The documentation and examples of using `<pre>` and `<code>` tags should remain, as we are still going to support it.

We should add `Code snippets` separately as new component.